metadoc-js/src/main/scala/metadoc/Metadoc.scala

```diff
@@ -20,7 +20,11 @@ object MetadocApp extends js.JSApp {
       indexBytes <- fetchBytes("metadoc.index")
       index = Index.parseFrom(indexBytes)
       bytes <- fetchBytes(
-        "semanticdb/" + index.files(0).replace(".scala", ".semanticdb")
+        "semanticdb/" +
+          index.files
+            .find(_.endsWith("Doc.scala"))
+            .get
+            .replace(".scala", ".semanticdb")
```

Comment:
We should probably add an `entrypoint` setting to the CLI and store it in metadoc.index for use here, unless we show some kind of directory browser on start.